RadixWallet/Core/DesignSystem/Components/AssetIcon.swift

```diff
@@ -27,6 +27,7 @@ public struct AssetIcon: View {
 	public var body: some View {
 		image
 			.resizable()
+			.scaledToFit()
```

Comment:
Reason why system images were looking bad on Debug Settings 🙂

Reply:
| Before | After |
| -- | -- |
| ![IMG_0015](https://github.com/radixdlt/babylon-wallet-ios/assets/164921079/db8ad8f4-fa40-4e33-9c9d-abf94269d023) | ![IMG_0014](https://github.com/radixdlt/babylon-wallet-ios/assets/164921079/36d7c152-a630-41bf-aee3-7d4872a452f5) |

Reply:
Wow that was ugly, good catch. I wonder if it's been like that for long.